docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,187 @@
+          - set has_fault to true: `has_fault = true`
+        - the supply water temperature should be <= 180F: `if shw_eq.setpoint_temperature > 180:`
+          - add to the rule note: `rule_note += "Based on the size categories for gas storage water heaters in ASHRAE 90.1 Table 7.8, gas storage water heaters with an input capacity between 75,000 and 105,000 btu/hr shall have a setpoint temperature of 180F.  The setpoint temperature of this water heater is " + shw_eq.setpoint_temperature + "F.  "; rule_status = FAIL`
+          - set has_fault to true: `has_fault = true`
+        - calculate the expected efficiency for VERY_SMALL draw pattern: `if draw_pattern == "VERY_SMALL": expected_UEF = 0.2674 – (0.0009 * volume)`
+        - calculate the expected efficiency for LOW draw pattern: `if draw_pattern == "LOW": expected_UEF = 0.5362 – (0.0012 * volume)`
+        - calculate the expected efficiency for MEDIUM draw pattern: `if draw_pattern == "MEDIUM": expected_UEF = 0.6002 – (0.0011 * volume)`
+        - calculate the expected efficiency for HIGH draw pattern: `if draw_pattern == "HIGH": expected_UEF = 0.6597 – (0.0009 * volume)`
+        - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+          - check if the actual uniform_energy_factor equal to the expected: `if uniform_energy_factor == expected_UEF and !has_faults:`
+            - set rule_status to "PASS": `rule_status = "PASS"; CONTINUE TO RULE ASSERTION`
+        - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note += "  Uniform Energy Factor is required but not provided"; CONTINUE TO RULE ASSERTION`
+      - otherwise the capacity is > 105,000 btu/hr: `else:`
+        - the capacity per volume should be < 4000 btu/hr/gallon: `if capacity_per_volume >= 4000:`
+          - add to the rule note: `rule_note += "Based on the size categories for gas storage water heaters in ASHRAE 90.1 Table 7.8, gas storage water heaters with an input capacity greater than 105,000 btu/hr shall have a capacity per volume of 4,000 btu/hr/gallon. This water heater has a capacity per gallon of " + capacity_per_volume + " btu/hr/gallon.  Consequently, this rule was not able to be assesed for this service water heater.  "; rule_status = UNDETERMINED`
+          - set has_fault to true: `has_fault = true`
+        - calculate the expected_thermal_efficiency : `expected_thermal_efficiency = 0.8`
+        - calculate max_standby_loss (btu/hr): `max_standby_loss = (input_capacity/800 + 110 * sqrt(volume))`
+        - make sure that the thermal efficiency is not NULL: `if thermal_efficiency != NULL:`
+          - make sure that the standy loss is not NULL: `if swh_eq.standby_loss_fraction != NULL:`
```

Comment:
agree this should be `STANDBY_LOSS_ENERGY`? not fraction?

Reply:
yes